marble-table/src/test/java/org/apache/calcite/table/TypeConvertUtilTest.java

```diff
@@ -0,0 +1,158 @@
+  }
+
+  @Test
+  public void testCastString() {
+    Assert.assertEquals("Bar", TypeConvertUtil.castString("Bar"));
+  }
+
+  @Test
+  public void testToBigDecimal() {
+    Assert.assertNull(TypeConvertUtil.toBigDecimal(null));
+
+    Assert.assertEquals(new BigDecimal(8), TypeConvertUtil.toBigDecimal(8));
+  }
+
+  @Test
+  public void testToBoolean() {
+    Assert.assertNull(TypeConvertUtil.toBoolean(null));
+
+    Assert.assertTrue(TypeConvertUtil.toBoolean(9));
+  }
+
+  @Test
+  public void testToDate() {
+    Assert.assertNull(TypeConvertUtil.toDate(null, null));
+
+    Assert.assertEquals("1970-01-25",
+            TypeConvertUtil.toDate(2147483647, null).toString());
+    Assert.assertEquals("2019-06-11",
+            TypeConvertUtil.toDate(1560265977787L, null).toString());
+    Assert.assertEquals("2019-06-11",
+            TypeConvertUtil.toDate(new Date(1560265977787L), null).toString());
+  }
+
+  @Test
+  public void testToDouble() {
+    Assert.assertNull(TypeConvertUtil.toDouble(null));
+    Assert.assertEquals(10.0, TypeConvertUtil.toDouble(10.0), 0.0);
+  }
+
+  @Test
+  public void testToFloat() {
+    Assert.assertNull(TypeConvertUtil.toFloat(null));
+
+    Assert.assertEquals(11.0f, TypeConvertUtil.toFloat(11.0f), 0.0f);
+  }
+
+  @Test
+  public void testToInteger() {
+    Assert.assertNull(TypeConvertUtil.toInteger(null));
+
+    Assert.assertEquals(new Integer(0), TypeConvertUtil.toInteger(false));
+    Assert.assertEquals(new Integer(1), TypeConvertUtil.toInteger(true));
+    Assert.assertEquals(new Integer(12), TypeConvertUtil.toInteger(12));
+  }
+
+  @Test
+  public void testToLong() {
+    Assert.assertNull(TypeConvertUtil.toLong(null));
+
+    Assert.assertEquals(new Long(13L), TypeConvertUtil.toLong(13));
+  }
+
+  @Test
+  public void testToShort() {
+    Assert.assertNull(TypeConvertUtil.toShort(null));
+
+    Assert.assertEquals(new Short((short)14), TypeConvertUtil.toShort(14));
+  }
+
+  @Test
+  public void testToString() {
+    Assert.assertNull(TypeConvertUtil.toString(null));
+
+    final NlsString nlsString = new NlsString("abc", null, null);
+    Assert.assertEquals("abc", TypeConvertUtil.toString(nlsString));
+    Assert.assertEquals("15", TypeConvertUtil.toString("15"));
+  }
+
+  @Test
+  public void testToTimestamp() {
+    Assert.assertNull(TypeConvertUtil.toTimestamp(null, null));
+
+    Assert.assertEquals("2019-06-11 16:12:57.787",
+            TypeConvertUtil.toTimestamp(1560265977787L,
+                    null).toString());
+    Assert.assertEquals("1970-01-25 21:31:23.647",
+            TypeConvertUtil.toTimestamp(2147483647,
+                    null).toString());
+    Assert.assertEquals("2019-06-11 16:12:57.787",
+            TypeConvertUtil.toTimestamp(new Timestamp(1560265977787L),
+                    null).toString());
+    Assert.assertEquals("2019-06-11 16:12:57.787",
+            TypeConvertUtil.toTimestamp(new Date(1560265977787L),
+                    null).toString());
+  }
+
```

Comment:
Remove blank line

Reply:
I usually put a blank line between the end of the last test method and the end of the class to match the blank line before the start of the first test. This is also the style in other test files in this project.